.github/ISSUE_TEMPLATE.md

```diff
@@ -1,8 +1,6 @@
-## What type of issue would you like to report?
+## Are you reporting feature or a bug?
 
-- [ ] Bug
-- [ ] Feature request
-- [ ] Other, please explain: …
+<!-- Put your issue type here.  -->
```

Comment:
Maybe, similar to PR template?
`<!-- Put your issue type here: Bug / Feature request / Other, please explain  -->`